src/document/doc-debug.js

```diff
@@ -167,7 +167,12 @@ function printDocToDebug(doc) {
           .join(",")}]${options})`;
       }
 
-      return `group(${printDoc(doc.contents)}${options})`;
+      return (
+        (doc.break ? "wrappedGroup" : "group") +
+        // [prettierx] --paren-spacing option (...)
+        (doc.addedLine ? "WithTrailingLine" : "") +
+        `(${printDoc(doc.contents)}${options})`
+      );
```

Comment:
Considering that this is for debugging, I am wondering if we should just do what they did in Prettier 2.3.0?